schema.sql

```diff
@@ -62,3 +62,6 @@ ANIMAL_ID INT,
 VET_ID INT,
 DATE_OF_VISIT DATE
 );
+
+
+CREATE INDEX animal_index ON visits(animal_id ASC);
```

Comment:
- [ ] Great job so far creating an index on visits for the (animal_id), Kindly do so for the visits on (vet_id) and for owners on(email). This is the most basic way of improving efficiency by indexing all foreign keys that are frequently queried.
Check the screenshot below
![image](https://user-images.githubusercontent.com/81932478/185015367-0d0dce04-a3af-4055-a00d-ba1c62f60896.png)